configman/config_manager.py

```diff
@@ -45,21 +45,50 @@
 import functools
 import warnings
 
-import configman as cm
-import converters as conv
-import config_exceptions as exc
-import value_sources
-import def_sources
-
 #==============================================================================
 # for convenience define some external symbols here - some client modules may
 # import these symbols from here rather than their origin definition location.
 # PyFlakes may erroneously flag some of these as unused
-from option import Option, Aggregation
-from dotdict import DotDict, DotDictWithAcquisition, iteritems_breadth_first
-from namespace import Namespace
-from required_config import RequiredConfig  # used elsewhere - do not remove
-from config_file_future_proxy import ConfigFileFutureProxy
+from configman import (
+    command_line
+)
+from configman.converters import (
+    to_string_converters,
+)
+from configman.config_exceptions import (
+    NotAnOptionError,
+)
+from configman.config_file_future_proxy import (
+    ConfigFileFutureProxy
+)
+from configman.def_sources import (
+    setup_definitions,
+)
+from configman.dotdict import (
+    DotDict,
+    DotDictWithAcquisition,
+    iteritems_breadth_first
+)
+from configman.environment import (
+    environment
+)
+from configman.namespace import (
+    Namespace
+)
+from configman.option import (
+    Option,
+    Aggregation
+)
+from configman.required_config import (
+    RequiredConfig  # not used directly in this file, but made available as
+                    # a type to be imported from this module
+)
```

Comment:
As a non-nit-but-me-just-sharing,
Just like I use brackets for imports that break the 80char rule, I use proper stylistics on comments too if they get too long
```
# initialize it to something 
var = 1
# When the comment is so long that it needs to be on more than one line it 
# becomes hard to see where it starts and where it ends so I capitalize
# and end on a full stop.
var *= 2
```